chapters/revisions.tex

```diff
@@ -1201,21 +1201,21 @@ \subsection{Main changes in Modelica 3.2}\doublelabel{main-changes-in-modelica-3
 
   \begin{itemize}
   \item
-    A new specialized class ``\lstinline!operator record!'' is introduced -- with
+    A new specialized class \lstinline!operator record! is introduced -- with
     specialized typing rules (the type is identified by the class name;
     all other Modelica classes have a structural type system where the
     type is only defined by the public elements). Overloaded operators
-    can only be defined inside an ``\lstinline!operator record!''. This change fixes
+    can only be defined inside an \lstinline!operator record!. This change fixes
     a flaw in Modelica 3.1, since the look-up of overloaded operators is
     performed by the record class name.
   \item
-    Inheritance of an ``operator record'' is allowed if defined via a
+    Inheritance of an \lstinline!operator record! is allowed if defined via a
     short class definition. This removes a restriction of operator
     overloading in Modelica 3.1, e.g., to define derived classes with
     units for the record elements, like deriving ComplexVoltage from
     Complex.
   \item
-    New overloaded element `0' in order that operator record classes can
+    New overloaded element `\lstinline!0!' in order that operator record classes can
```

Comment:
Similary comment about preferring \lstinline!'0'!

Reply:
Right, this was fixed together with the previous case of the same error.